src/valid/function.rs

```diff
@@ -143,6 +143,10 @@ pub enum FunctionError {
         Handle<crate::Expression>,
         UniformityDisruptor,
     ),
+    #[error("Argument '{name}' at index {index} defines a pipeline input for a regular function")]
```

Comment:
This is a statement of fact, but it doesn't explain what's wrong. Suggest phrasing this as:

> Functions that are not entry points cannot have `@location` or `@builtin` attributes on their arguments

Reply:
I was having issues with the phrasing, thanks for the suggestion